### PR TITLE
MGMT-2423 Display cluster/host UUIDs

### PR DESCRIPTION
**Pull request by jtomasek**
Display Cluster UUID in cluster properties
Display Host UUID in host expanded detail

Screenshot from 2020-10-13 11-04-25
Screenshot from 2020-10-13 11-05-24

### CODE REVIEW

**Comment by nirfarkas** [comment]
Nit:
Reviewing both screenshots I wonder what is the convention we should use in the “Details” section:
In the cluster details, we show "Cluster network CIDR" & "Cluster network Host prefix" but in the host details, we're not adding "host" to any of the headers...
So, should it be "Cluster UUID" just "UUID"?

**Comment by jtomasek** [comment]
"Cluster network CIDR" and "Cluster network Host prefix" are specific OpenShift terms that's why it includes words 'Cluster and Host'. (for example compare 'Cluster network CIDR' vs 'Service network CIDR')

File: src/components/clusterDetail/ClusterProperties.tsx [hunk 1]
```diff
@@ -14,21 +14,26 @@ const ClusterProperties: React.FC<ClusterPropertiesProps> = ({ cluster }) => (
...
+    <GridItem md={6} lg={4} xl={4}>
       <DetailList>
         <DetailItem title="Cluster network CIDR" value={cluster.clusterNetworkCidr} />
         <DetailItem title="Cluster network host prefix" value={cluster.clusterNetworkHostPrefix} />
         <DetailItem title="Service network CIDR" value={cluster.serviceNetworkCidr} />
       </DetailList>
     </GridItem>
+    <GridItem md={6} lg={4} xl={4}>
+      <DetailList>
+        <DetailItem title="UUID" value={cluster.id} />
```
Comment (mareklibra) [inline src/components/clusterDetail/ClusterProperties.tsx:34]: We need to make sure, the user understands that this is just the `Assisted Installed UUID` and not the OCP ID which the user will use any farther.
There is a plan/wish to expose this second (probably more important for the user) ID too, but it's so far missing in the API.

@nirfarkas @andybraren

**Review by mareklibra — CHANGES_REQUESTED** [review]

File: src/components/clusterDetail/ClusterProperties.tsx (hunk 1 above)
Comment (jtomasek) [reply src/components/clusterDetail/ClusterProperties.tsx:34]: I've discussed this briefly with @andybraren yesterday. I am calling the ID just UUID to imply that it is the cluster resource ID and when it is available, we could call the OCP ID 'OpenShift Cluster ID'. This should provide enough differentiation, what do you think?

File: src/components/clusterDetail/ClusterProperties.tsx [hunk 2]
```diff
@@ -14,21 +14,26 @@ const ClusterProperties: React.FC<ClusterPropertiesProps> = ({ cluster }) => (
...
     </GridItem>
-    <GridItem md={6} lg={6} xl={6}>
+    <GridItem md={6} lg={4} xl={4}>
       <DetailList>
         <DetailItem title="Cluster network CIDR" value={cluster.clusterNetworkCidr} />
         <DetailItem title="Cluster network host prefix" value={cluster.clusterNetworkHostPrefix} />
         <DetailItem title="Service network CIDR" value={cluster.serviceNetworkCidr} />
       </DetailList>
     </GridItem>
+    <GridItem md={6} lg={4} xl={4}>
```
[the remark this answers is not in the document]
Comment (jtomasek) [reply src/components/clusterDetail/ClusterProperties.tsx]: done

File: src/components/clusterDetail/ClusterProperties.tsx (hunk 1 above)
Comment (mareklibra) [reply src/components/clusterDetail/ClusterProperties.tsx]: We can call those who understand the difference "Advanced Assisted Installer" users.

**Review by mareklibra — APPROVED** [review]

**Pull request merged**